py3dtilers/Common/tileset_creation.py

```diff
@@ -105,6 +105,9 @@ def create_tile_content(objects, extension_name=None, with_texture=False):
     ids = [feature.get_id() for feature in objects]
     bt = BatchTable()
     bt.add_property_from_array("id", ids)
+    if store_alt_id:
+        alt_ids = [feature.get_alt_id() for feature in objects]
+        bt.add_property_from_array("alt_id", alt_ids)
```

Comment:
When creating levels of details, only the tiles with the detailled geometries (those with the cityGML geometries in this case) will have alt ids. The tiles containing the LODs will have an alt id set to 'None', and thus will store a list of 'None' in the batch table.
Maybe we could check the value of the alt ids, and then the 'store_alt_id' boolean would become useless:
```
bt.add_property_from_array("id", ids)

alt_ids = [feature.get_alt_id() for feature in objects if feature.get_alt_id() is not None]
if alt_ids:
     bt.add_property_from_array("alt_id", alt_ids)
```